live-build/hooks/30-fix-timedatectl.chroot

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+mv /usr/bin/timedatectl /usr/bin/timedatectl.real
+
+cat >/usr/bin/timedatectl<<EOF
+#!/bin/sh
+
+# wrapper to make sure timedatectl works with the
+# writable-paths setup on ubuntu core (making the symlink
+# absolute instead of using a relative path)
+# LP: #1650688 for details
+
+set -e
+
+TIMEDATECTL=/usr/bin/timedatectl.real
+
+case $1 in
```

Comment:
Maybe not relevant here because of the inputs we get, but don't we want quotes around $1 and $2 ?